packages/zowe-explorer/src/utils/AuthUtils.ts

```diff
@@ -179,6 +179,125 @@ export class AuthUtils {
         sessionNode.setProfileToChoice(profile);
         try {
             const commonApi = getCommonApi(profile);
+            const usingBasicAuth = profile.profile.user && profile.profile.password;
+            const usingCertAuth = profile.profile.certFile && profile.profile.certKeyFile;
+            let usingTokenAuth: boolean;
+            try {
+                usingTokenAuth = await AuthUtils.isUsingTokenAuth(profile.name);
+            } catch (err) {
+                ZoweLogger.error(err);
+            }
+            const toolTipList = sessionNode.tooltip === "" ? [] : (sessionNode.tooltip as string).split("\n");
+
+            const authMethodIndex = toolTipList.findIndex((key) => key.startsWith("Auth Method: "));
+            if (authMethodIndex === -1) {
+                switch (true) {
+                    case Boolean(usingTokenAuth): {
+                        toolTipList.push(`Auth Method: Token-based Authentication`);
+                        break;
+                    }
+                    case Boolean(usingBasicAuth): {
+                        toolTipList.push(`Auth Method: Basic Authentication`);
+                        toolTipList.push(`User: ${profile.profile.user as string}`);
+                        break;
+                    }
+                    case Boolean(usingCertAuth): {
+                        toolTipList.push(`Auth Method: Certificate Authentication`);
+                        break;
+                    }
+                    default: {
+                        toolTipList.push(`Auth Method: Unknown`);
+                        break;
+                    }
+                }
+            } else {
+                switch (true) {
+                    case Boolean(usingTokenAuth): {
+                        toolTipList[authMethodIndex] = `Auth Method: Token-based Authentication`;
+                        break;
+                    }
+                    case Boolean(usingBasicAuth): {
+                        toolTipList[authMethodIndex] = `Auth Method: Basic Authentication`;
+                        const userIDIndex = toolTipList.findIndex((key) => key.startsWith("User: "));
+                        if (userIDIndex !== -1) {
+                            toolTipList[userIDIndex] = `User: ${profile.profile.user as string}`;
+                        } else {
+                            toolTipList.splice(authMethodIndex + 1, 0, `User: ${profile.profile.user as string}`);
+                        }
+                        break;
+                    }
+                    case Boolean(usingCertAuth): {
+                        toolTipList[authMethodIndex] = `Auth Method: Certificate Authentication`;
+                        break;
+                    }
+                    default: {
+                        toolTipList[authMethodIndex] = `Auth Method: Unknown`;
```

Comment:
Could this logic be simplified to avoid duplication? It seems like there is a good amount of overlap with these strings and these strings are hardcoded multiple times.

Also, I think we should localize these strings to support future translation.